cms/migrations/0022_auto_20180620_1551.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.1b1 on 2018-06-20 15:51
+
+from django.db import migrations, models
+
+from cms.utils.compat import DJANGO_2_0
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0021_auto_20180507_1432'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='page',
+            options={'default_permissions': ('add', 'change', 'delete'), 'permissions': (('view_page', 'Can view page'), ('publish_page', 'Can publish page'), ('edit_static_placeholder', 'Can edit static placeholders')), 'verbose_name': 'page', 'verbose_name_plural': 'pages'},
+        ),
+    ]
+    if not DJANGO_2_0:
```

Comment:
why is this needed on django >= 2.1?

Reply:
When running tests against dj21 it's showing missing migration